frontend-react/src/TelemetryService.ts

```diff
@@ -43,3 +43,13 @@ const createTelemetryService = () => {
 };
 
 export const ai = createTelemetryService();
+
+export function getAppInsightsHeaders(): { [key: string]: string } {
+    return {
+        "x-ms-session-id": getAppInsightsSessionId(),
```

Comment:
To test, I think we can make a mock of `getAppInsightsSessionId()` and hijack the value, then test through `msw`'s request object, or just object drilling to find it in the created fetch config.